### DISCUSSION TITLE
Can't find EBITDA data that matches yahoo finance site

### DISCUSSION THREAD

**Discussion by AbdulHassan0** [ranaroussi/yfinance#1889] [Yahoo's data]
Whenever I try to get ebitda data for a specific ticker it returns a number that doesn't match the website.  An example as of 1/4/23 would be:

**msft = yf.Ticker("MSFT")**

**msft.info['ebitda']**

**98841001984**

The yahoo finance website shows the number as: 

![image](https://user-images.githubusercontent.com/106120256/210655229-8d4fd5ea-214e-44fa-8b5d-845318b5591c.png)

yfinance version = 0.2.3

**Comment by rickturner2001** [comment]
@AbdulHassan0 

![temp](https://user-images.githubusercontent.com/93800987/212849270-106a2e89-d838-4ab5-a7b4-1eac08766980.jpg)


source: https://finance.yahoo.com/quote/MSFT/key-statistics/

when I run the following snippet:

```python3
import yfinance as yf

msft = yf.Ticker("MSFT")
print(msft.info)
```



```python3
ebitda: 98841001984
```

what is your source for that data?

**Comment by AbdulHassan0** [comment]
The source/screenshot was the income statement screen: https://finance.yahoo.com/quote/msft/financials/

I appreciate the clarification though, thank you!  Do you know what time period the $98.84B is referring to?
